.github/workflows/rust.yml

```
@@ -165,6 +165,11 @@ jobs:
         run: |
           forest-cli snapshot export
           sha256sum -c *.sha256sum
+      - name: validate snapshot
+        run: |
+          mainnet_with_calibnet_snapshot_should_fail=$(forest-cli --chain mainnet snapshot validate $SNAPSHOT_DIRECTORY/*.car --force)
+          if [ "$?" -eq 0 ]; then echo "calibnet snapshot validation on mainnet should fail" exit 1; fi
```

Github runs bash with the `-e` flag which causes it to exit immediately on errors (unless the commands are part of an if-statement). You probably want something like:

```suggestion
          if forest-cli --chain mainnet snapshot validate $SNAPSHOT_DIRECTORY/*.car --force; then
              echo "calibnet snapshot validation on mainnet should fail";
              exit 1;
          fi
```